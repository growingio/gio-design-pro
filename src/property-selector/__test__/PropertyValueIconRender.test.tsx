import React from 'react';
import { render } from '@testing-library/react';
import PropertyValueIconRender from '../PropertyValueIconRender';

const groupList: string[] = [
  'cs',
  'ads',
  'page',
  'event',
  'item',
  'app',
  'people',
  'visitor',
  'element',
  'conversion',
  'user',
  'tag',
  'geo',
  'device',
  'origin',
];
describe('PropertyValueIconRender', () => {
  it('can render the icon by groupid', async () => {
    const { container } = render(
      <>
        {groupList.map((g: string) => (
          <PropertyValueIconRender key={`idx-${g}`} group={g} />
        ))}
      </>
    );
    expect(container).toMatchSnapshot();
  });
  it('will be render empty when group id is not in case', async () => {
    const { container } = render(<PropertyValueIconRender group="not exist group id" />);
    expect(container.querySelector('svg')).toBeNull();
  });
});
