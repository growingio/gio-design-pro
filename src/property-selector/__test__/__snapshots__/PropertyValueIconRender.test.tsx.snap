// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyValueIconRender can render the icon by groupid 1`] = `
<div>
  <span
    aria-label="event-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M2.601 12A.602.602 0 012 11.398V2.601A.6.6 0 012.601 2h5.797C8.73 2 9 2.269 9 2.601v8.608c0 .285.073.552.191.791h-6.59zM11.5 4h-1.262A.238.238 0 0110 3.761V2.5A1.5 1.5 0 008.5 1h-6A1.5 1.5 0 001 2.5v9A1.5 1.5 0 002.5 13h9a1.5 1.5 0 001.5-1.5v-6A1.5 1.5 0 0011.5 4zm-.145 8h-.71a.645.645 0 01-.645-.645v-5.71c0-.356.29-.645.644-.645h.711c.355 0 .645.289.645.645v5.71c0 .356-.29.645-.645.645zM3.5 3.5h4A.5.5 0 018 4v2a.5.5 0 01-.5.5h-4A.5.5 0 013 6V4a.5.5 0 01.5-.5zm.3 2.2h3.4V4.3H3.8v1.4zm3.7 1.8h-4a.5.5 0 000 1h4a.5.5 0 000-1zm-4 2h4a.5.5 0 010 1h-4a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="event-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M2.601 12A.602.602 0 012 11.398V2.601A.6.6 0 012.601 2h5.797C8.73 2 9 2.269 9 2.601v8.608c0 .285.073.552.191.791h-6.59zM11.5 4h-1.262A.238.238 0 0110 3.761V2.5A1.5 1.5 0 008.5 1h-6A1.5 1.5 0 001 2.5v9A1.5 1.5 0 002.5 13h9a1.5 1.5 0 001.5-1.5v-6A1.5 1.5 0 0011.5 4zm-.145 8h-.71a.645.645 0 01-.645-.645v-5.71c0-.356.29-.645.644-.645h.711c.355 0 .645.289.645.645v5.71c0 .356-.29.645-.645.645zM3.5 3.5h4A.5.5 0 018 4v2a.5.5 0 01-.5.5h-4A.5.5 0 013 6V4a.5.5 0 01.5-.5zm.3 2.2h3.4V4.3H3.8v1.4zm3.7 1.8h-4a.5.5 0 000 1h4a.5.5 0 000-1zm-4 2h4a.5.5 0 010 1h-4a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="page-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M0 2a1 1 0 011-1h12a1 1 0 011 1v10a1 1 0 01-1 1H1a1 1 0 01-1-1V2zm1 2v8h12V4H1zm1-2a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 110 1 .5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="event-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M2.601 12A.602.602 0 012 11.398V2.601A.6.6 0 012.601 2h5.797C8.73 2 9 2.269 9 2.601v8.608c0 .285.073.552.191.791h-6.59zM11.5 4h-1.262A.238.238 0 0110 3.761V2.5A1.5 1.5 0 008.5 1h-6A1.5 1.5 0 001 2.5v9A1.5 1.5 0 002.5 13h9a1.5 1.5 0 001.5-1.5v-6A1.5 1.5 0 0011.5 4zm-.145 8h-.71a.645.645 0 01-.645-.645v-5.71c0-.356.29-.645.644-.645h.711c.355 0 .645.289.645.645v5.71c0 .356-.29.645-.645.645zM3.5 3.5h4A.5.5 0 018 4v2a.5.5 0 01-.5.5h-4A.5.5 0 013 6V4a.5.5 0 01.5-.5zm.3 2.2h3.4V4.3H3.8v1.4zm3.7 1.8h-4a.5.5 0 000 1h4a.5.5 0 000-1zm-4 2h4a.5.5 0 010 1h-4a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="item-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M12.088 4.665c.507-.479 1.207-.391 1.636.174a1.39 1.39 0 01-.194 1.872 1.124 1.124 0 01-1.207.175l-4.324 5.181c.079.173.117.391.117.609 0 .74-.506 1.307-1.168 1.307-.662 0-1.168-.567-1.168-1.307 0-.043.009-.087.019-.13a.618.618 0 00.019-.131l-2.765-1.829a1.15 1.15 0 01-.74.305c-.662 0-1.168-.566-1.168-1.306 0-.435.233-.871.545-1.089l-.662-3.265c-.43-.043-.857-.392-.974-.914C-.14 3.62.21 2.923.833 2.705c.39-.13.779-.043 1.09.262l2.766-1.829c.039-.305.155-.61.389-.827.506-.479 1.208-.392 1.636.174.156.218.234.436.273.697l5.1 3.483zm-.39.74L6.79 2.054a.776.776 0 01-.194.305l-.011.008a1.483 1.483 0 01-.34.209v2.308c.429.13.818.479 1.013.958l4.439-.306v-.13zM2.312 3.708l2.57-1.654c.02.022.03.043.04.065.01.022.019.044.038.065.157.174.313.305.506.392v2.308a1.344 1.344 0 00-.817.61L2.312 4.36c0-.072.005-.144.01-.217.008-.145.017-.29-.01-.436zm-.505 1.35l.662 3.265c.194 0 .35.087.506.174l1.44-1.263a1.99 1.99 0 01-.116-.696c0-.043.01-.086.02-.13a.61.61 0 00.019-.132L1.807 5.056zm4.44 6.678L3.48 9.845c.022-.147.022-.515-.04-.652l1.442-1.307c.234.218.546.35.896.35l.639 3.358-.171.142zm1.17-.14l4.281-5.189-4.322.262c-.039.652-.39 1.175-.858 1.436l.587 3.404c.09 0 .158.027.239.058.023.01.047.019.073.028z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="event-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M2.601 12A.602.602 0 012 11.398V2.601A.6.6 0 012.601 2h5.797C8.73 2 9 2.269 9 2.601v8.608c0 .285.073.552.191.791h-6.59zM11.5 4h-1.262A.238.238 0 0110 3.761V2.5A1.5 1.5 0 008.5 1h-6A1.5 1.5 0 001 2.5v9A1.5 1.5 0 002.5 13h9a1.5 1.5 0 001.5-1.5v-6A1.5 1.5 0 0011.5 4zm-.145 8h-.71a.645.645 0 01-.645-.645v-5.71c0-.356.29-.645.644-.645h.711c.355 0 .645.289.645.645v5.71c0 .356-.29.645-.645.645zM3.5 3.5h4A.5.5 0 018 4v2a.5.5 0 01-.5.5h-4A.5.5 0 013 6V4a.5.5 0 01.5-.5zm.3 2.2h3.4V4.3H3.8v1.4zm3.7 1.8h-4a.5.5 0 000 1h4a.5.5 0 000-1zm-4 2h4a.5.5 0 010 1h-4a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="page-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M0 2a1 1 0 011-1h12a1 1 0 011 1v10a1 1 0 01-1 1H1a1 1 0 01-1-1V2zm1 2v8h12V4H1zm1-2a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 110 1 .5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="event-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M2.601 12A.602.602 0 012 11.398V2.601A.6.6 0 012.601 2h5.797C8.73 2 9 2.269 9 2.601v8.608c0 .285.073.552.191.791h-6.59zM11.5 4h-1.262A.238.238 0 0110 3.761V2.5A1.5 1.5 0 008.5 1h-6A1.5 1.5 0 001 2.5v9A1.5 1.5 0 002.5 13h9a1.5 1.5 0 001.5-1.5v-6A1.5 1.5 0 0011.5 4zm-.145 8h-.71a.645.645 0 01-.645-.645v-5.71c0-.356.29-.645.644-.645h.711c.355 0 .645.289.645.645v5.71c0 .356-.29.645-.645.645zM3.5 3.5h4A.5.5 0 018 4v2a.5.5 0 01-.5.5h-4A.5.5 0 013 6V4a.5.5 0 01.5-.5zm.3 2.2h3.4V4.3H3.8v1.4zm3.7 1.8h-4a.5.5 0 000 1h4a.5.5 0 000-1zm-4 2h4a.5.5 0 010 1h-4a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="element-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M7 0a7 7 0 100 14A7 7 0 007 0zm0 1c3.309 0 6 2.691 6 6s-2.691 6-6 6-6-2.691-6-6 2.691-6 6-6zm-.503 5.008l.815 4.852s.602-.637 1.1-1.081l1.092 1.503c.044.061.14.065.214.01l.466-.35c.074-.056.098-.15.054-.211L9.174 9.269c.556-.23 1.47-.492 1.47-.492l-4.147-2.77zM3.97 9.164l1.213-.91c.187-.141.453-.1.598.096a.44.44 0 01-.074.614l-1.213.91c-.183.138-.45.086-.596-.11a.427.427 0 01.072-.6zM10.908 4.5a.422.422 0 00-.599-.096l-1.205.905a.43.43 0 00-.079.606.43.43 0 00.605.103l1.204-.905a.437.437 0 00.074-.613zM4.684 3.47a.42.42 0 01.595.092l.943 1.273a.428.428 0 01-.083.601.42.42 0 01-.588-.098L4.61 4.065a.428.428 0 01.075-.595zm2.975 1.605a.413.413 0 00.479-.36l.189-1.539a.455.455 0 00-.371-.501.413.413 0 00-.479.36l-.19 1.539a.456.456 0 00.372.5zM5.002 6.29c.23.042.393.263.37.501-.037.243-.249.4-.478.36l-1.519-.257a.436.436 0 01-.37-.5.413.413 0 01.478-.36l1.519.256z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="page-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M0 2a1 1 0 011-1h12a1 1 0 011 1v10a1 1 0 01-1 1H1a1 1 0 01-1-1V2zm1 2v8h12V4H1zm1-2a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 100 1 .5.5 0 000-1zm1.8 0a.5.5 0 110 1 .5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="user-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M11.821 9.273c.386.29 1.78 1.445 2.025 2.414.244.971.3 2.051-.63 2.313H.78c-.926-.261-.872-1.341-.626-2.312.244-.969 1.636-2.124 2.024-2.414.26-.193.662-.181 1.067-.169.205.006.411.013.6-.008.558-.059.956-.741 1.027-.866.073-.125.076-.601.065-.678-.01-.076-.103-.238-.103-.238-.38-.443-.817-1.532-.817-1.532-.779-1.015-.09-1.606-.09-1.606-.027-.139-.087-1.386-.038-1.671l.015-.089c.05-.314.139-.88.735-1.595C5.297.033 6.5 0 6.5 0l.5.001L7.5 0s1.203.031 1.862.821c.598.717.687 1.285.735 1.597l.014.087c.05.286-.009 1.533-.036 1.671 0 0 .69.591-.09 1.606 0 0-.436 1.089-.819 1.532 0 0-.092.162-.103.238-.01.076-.007.553.065.678.071.125.47.807 1.028.866.188.02.394.014.6.008.404-.012.807-.025 1.065.169zm-4.82 3.956c.542.012 5.305.014 5.714-.09.366-.092.31-.401.226-.851a10.158 10.158 0 01-.048-.27c-.095-.596-1.116-1.564-1.525-1.828-.267-.171-.412-.183-.777-.214a12.684 12.684 0 01-.814-.09C8.713 9.734 8.291 8.74 8.174 8.38c-.117-.36-.05-1.228.046-1.322.094-.093.49-.788.53-.902.041-.115.225-.546.225-.546.625-.748.292-1.099.236-1.136-.055-.038-.063-.287-.055-.766l.008-.236c.015-.353.027-.662-.115-1.46C8.876 1.04 7.361.82 7.361.82h-.723s-1.515.22-1.688 1.19c-.141.798-.13 1.107-.116 1.46.003.075.006.152.007.236.01.48 0 .73-.054.766-.054.037-.389.39.238 1.136 0 0 .183.431.225.545.04.115.435.81.53.903.094.094.163.961.045 1.322-.118.36-.539 1.355-1.6 1.507-.368.053-.624.075-.818.09-.364.031-.508.043-.775.214-.408.264-1.429 1.232-1.524 1.828-.015.096-.032.187-.048.273-.083.448-.14.756.224.848.411.104 5.174.102 5.716.09zm.158-3.623h-.312a.197.197 0 01-.17-.097l-.265-.437c0-.189.133-.34.372-.34h.437c.24 0 .379.152.379.34l-.271.439a.2.2 0 01-.17.095zm-.049 2.632l.48-.36c.06-.045.09-.12.075-.194L7.351 9.9a.202.202 0 00-.198-.165h-.296a.201.201 0 00-.198.164l-.332 1.786a.2.2 0 00.075.195l.466.358a.198.198 0 00.242.001z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="tag-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      clip-rule="evenodd"
      fill="currentColor"
      fill-rule="evenodd"
      height="16px"
      stroke-linejoin="round"
      stroke-miterlimit="2"
      viewBox="0 0 16 16"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.31.406a1.38 1.38 0 00-.976-.404H1.382a1.38 1.38 0 00-1.38 1.38v4.952c0 .367.144.718.404.978l8.285 8.285c.54.54 1.414.54 1.954 0l4.952-4.952a1.38 1.38 0 000-1.953L7.31.406zm-1.548.976c.367 0 .718.147.977.405 1.379 1.38 5.894 5.895 7.394 7.393.27.27.27.708 0 .977l-3.976 3.976a.691.691 0 01-.978 0c-1.498-1.5-6.013-6.015-7.393-7.393a1.385 1.385 0 01-.404-.977v-3.69c0-.38.31-.69.691-.69h3.69z"
        fill="currentColor"
      />
      <path
        d="M5.144 3.43a1.715 1.715 0 11-.001 3.43 1.715 1.715 0 01.001-3.43zm0 1.143a.573.573 0 00-.571.572c0 .314.257.571.571.571a.573.573 0 00.571-.571.573.573 0 00-.57-.572z"
        fill="currentColor"
      />
    </svg>
  </span>
  <span
    aria-label="geo-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M6.998 0C3.14 0 0 3.137 0 6.991c0 3.852 3.14 6.986 6.998 6.986s6.998-3.134 6.998-6.986C13.996 3.137 10.856 0 6.998 0zM5.922 1.032A8.126 8.126 0 004.03 3.62a5.558 5.558 0 01-1.334-.893 6.116 6.116 0 013.226-1.695zM2.26 3.212c.267.251.775.664 1.522 1.021a8.049 8.049 0 00-.459 2.432H.943A5.996 5.996 0 012.26 3.212zm-1.317 4.1h2.38c.031.848.185 1.667.459 2.433a6.156 6.156 0 00-1.522 1.019A5.988 5.988 0 01.943 7.312zm1.753 3.939a5.538 5.538 0 011.334-.894 8.049 8.049 0 001.892 2.587 6.1 6.1 0 01-3.226-1.693zm3.977 1.473a7.608 7.608 0 01-2.03-2.625 6.796 6.796 0 012.03-.398v3.023zm0-3.673a7.42 7.42 0 00-2.28.439 7.27 7.27 0 01-.421-2.178h2.701v1.739zm0-2.386H3.972a7.239 7.239 0 01.421-2.176c.71.259 1.477.406 2.28.436v1.74zm0-2.39a6.826 6.826 0 01-2.03-.395 7.615 7.615 0 012.03-2.627v3.022zm6.38 2.39h-2.378a8.088 8.088 0 00-.46-2.432 6.098 6.098 0 001.522-1.021 6.013 6.013 0 011.316 3.453zM11.3 2.726a5.536 5.536 0 01-1.335.893 8.13 8.13 0 00-1.891-2.588A6.116 6.116 0 0111.3 2.726zM7.323 1.253a7.66 7.66 0 012.03 2.627 6.845 6.845 0 01-2.03.395V1.253zm0 3.672a7.46 7.46 0 002.28-.436c.249.692.389 1.421.42 2.176h-2.7v-1.74zm0 2.387h2.7a7.293 7.293 0 01-.42 2.178 7.42 7.42 0 00-2.28-.439V7.312zm0 5.412V9.701a6.747 6.747 0 012.03.398 7.632 7.632 0 01-2.03 2.625zm.751.218a8.072 8.072 0 001.891-2.586c.66.323 1.102.68 1.335.894a6.077 6.077 0 01-3.226 1.692zm3.661-2.178a6.193 6.193 0 00-1.522-1.019 8.138 8.138 0 00.461-2.433h2.377a5.98 5.98 0 01-1.316 3.452z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="device-property-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M5.576 11h3a.5.5 0 010 1h-3a.5.5 0 010-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
      <path
        clip-rule="evenodd"
        d="M10.076 0h-6a2 2 0 00-2 2v10a2 2 0 002 2h6a2 2 0 002-2V2a2 2 0 00-2-2zm0 1a1 1 0 011 1v10a1 1 0 01-1 1h-6a1 1 0 01-1-1V2a1 1 0 011-1h1.5l.361.724A.502.502 0 006.385 2h1.382a.502.502 0 00.448-.276L8.576 1h1.5zm-1.5 10h-3a.5.5 0 000 1h3a.5.5 0 000-1z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
  <span
    aria-label="user-origin-outlined"
    class="gio-icon"
    role="img"
  >
    <svg
      fill="currentColor"
      height="16px"
      viewBox="0 0 14 14"
      width="16px"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clip-rule="evenodd"
        d="M7 0a7 7 0 100 14A7 7 0 007 0zm0 1c3.309 0 6 2.691 6 6s-2.691 6-6 6-6-2.691-6-6 2.691-6 6-6zm3.084 3.575L8.132 7.617a.127.127 0 01-.175.04.12.12 0 01-.026-.025L6.439 5.917a.126.126 0 01.013-.178.137.137 0 01.029-.02l3.286-1.51a.253.253 0 01.317.366zM3.904 9.72a.252.252 0 00.243.017l3.285-1.51a.126.126 0 00.042-.197L5.983 6.314a.126.126 0 00-.201.014L3.83 9.371a.25.25 0 00.075.348z"
        fill="currentColor"
        fill-rule="evenodd"
      />
    </svg>
  </span>
</div>
`;
